File: stripe_android/android/src/main/kotlin/com/facebook/react/bridge/BaseActivityEventListener.java
package com.facebook.react.bridge;

import android.content.Intent;

import io.flutter.plugin.common.PluginRegistry;

public abstract class BaseActivityEventListener implements ActivityEventListener, PluginRegistry.ActivityResultListener {
    @Override
    public boolean onActivityResult(int requestCode, int resultCode, Intent data) {
        onActivityResult(null, requestCode, resultCode, data);
        return true;
    }
}

File: stripe_android/android/src/main/kotlin/com/facebook/react/bridge/BaseActivityEventListener.kt
package com.facebook.react.bridge

import android.content.Intent
import com.stripe.android.Stripe
import io.flutter.plugin.common.PluginRegistry.ActivityResultListener

abstract class BaseActivityEventListener : ActivityEventListener, ActivityResultListener {
    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?): Boolean {
        return onActivityResult(null, requestCode, resultCode, data)
    }
}